devlogs/mayeesha/daily-logs/2025-03-17.md

```
@@ -0,0 +1,92 @@
+    `ElevatedButton(
+    onPressed: () {
+    context.go('/details/HelloFlutter'); // Passing "HelloFlutter" to Details Page
+     },
+    child: Text('Go to Details Page with Data'),
+    );`
+
+🔹 context.go() is used to navigate to a new page.
+🔹 context.push() is used to push a new page on top of the current page (thus, back button works).
+🔹 context.pop() is used to go back to the previous page.
+
+# ref in Riverpod
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)